docs/pages/application-access/enroll-kubernetes-applications/reference.mdx

```diff
@@ -108,7 +108,9 @@ annotations:
     - "localhost"
     - "jenkins.internal.dev"
     headers:
-    - "X-Custom-Header: example"
-    - "Authorization: Bearer {{internal.jwt}}"
+    - name: "X-Custom-Header"
```

Comment:
This no longer matches the link above, so the _same as one would put into rewrite config section_ text is not true and needs to be updated.

Reply:
> This no longer matches the link above, so the _same as one would put into rewrite config section_ text is not true and needs to be updated.

[Dynamic registration syntax](https://goteleport.com/docs/application-access/guides/connecting-apps/#rewrite-redirect) tab already has the right syntax for header rewriting. Is that what you mean?

Reply:
Yeah, but the dynamic registration syntax isn't selected by default, so if someone follows the link here and takes what they see on screen and dumps it into the `app-rewrite` annotation, they're likely going to make a mistake.

Maybe we just need to say something like _same as one would use when configuring an app with dynamic registration._

Reply:
> Yeah, but the dynamic registration syntax isn't selected by default, so if someone follows the link here and takes what they see on screen and dumps it into the `app-rewrite` annotation, they're likely going to make a mistake.
> 
> Maybe we just need to say something like _same as one would use when configuring an app with dynamic registration._

Added